doc/proposals/improved-scorecard-config.md

```diff
@@ -0,0 +1,112 @@
+# Improved Scorecard Config
+
+Implementation Owner: AlexNPavel
+Status: Draft
+
+- [Background](#Background)
+- [Goals](#Goals)
+- [Design overview](#Design-overview)
+- [User facing usage](#User-facing-usage)
+
+## Background
+
+The `scorecard` subcommand in the `operator-sdk` currently has ~15 different command line flags. This creates
+unnecessary complexity for users. Also, many of the config options only apply to the internal plugins, and
+this may become confusing to users who use external plugins, as they may think that these config options
+affect their external plugins. Using a config file that configures plugins on an individual
+basis can help clarify these differences as well as make configuration a lot cleaner
+
+## Goals
+
+Drastically reduce the amount of command line flags in the `scorecard` subcommand and define a configuration file
+format that can allow for the simple configuration of both internal and external plugins.
+
+## Design overview
+
+The configuration for the scorecard will be in a `scorecard` subsection of the config file. This will allow
+the config file to continue working properly when the SDK adds global config file support for all subcommands.
+Under the scorecard subsection there are configuration options that apply to the entire scorecard as well as
+a section that allows for the configuration of both internal and external plugins. Here are the config options
+and what they do:
+
+- `kubeconfig` string - path to kubeconfig. This option sets the kubeconfig for internal plugins and sets the `KUBECONFIG` env var for external plugins
+- `output` string - sets output format. Valid values: `human-readable` and `json`
+- `plugin-dir` string - path to scorecard plugin directory. This is the directory where the plugins are run from, and all executable files in a `bin` subdirectory of the `plugin-dir` are automtically run by default.
+- `plugins` - an array of objects that configure both internal and external scorecard plugins
+
+The objects in the `plugins` configuration have 3 elements: `name`, `disabled`, and either `internal` or `external`. The `name` is
+the name of the plugin and `internal` and `external` are configuration blocks used if the plugin in `internal` or `external`
+respectively. If `internal` and `external` are both specified for the same block, the plugin is automatically marked as failed. The
+`disabled` field is `false` by default, but can be set to `true` to disable tests that would be automatically run, such as the
```

Comment:
> `disabled` field is `false` by default, but can be set to `true` to disable tests

this reads a bit confusing to me, `disable` would make a bit more sense to me. WDYT?

Reply:
Yeah, that would be a bit better